src/renderer/css/components/AddServerModal.css

```diff
@@ -0,0 +1,181 @@
+html {
+  height: 100%;
+}
+body {
+  min-height: 100%;
+}
+
+.NewTeamModal-noBottomSpace {
+  padding-bottom: 0px;
+  margin-bottom: 0px;
+}
+
+.modal {
+  background-color: white;
+  color: #333;
+  border-radius: 4px;
+  border: 1px solid;
+  border-color: #666;
+  padding: 5px;
+  display: block;
+  font-size: 14px;
+  line-height: 1.4;
+  width: 1000px;
+  position: absolute;
+  top: 25%;
+  left: 50%;
+  transform: translate(-50%, -25%);
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+form {
```

Comment:
can we avoid non-class styling? This gives us a lot of trouble especially in webapp when adding new features and the styles overtake, since they are element specific?